src/components/common/Layout.js

```diff
@@ -51,17 +51,20 @@ const DefaultLayout = ({ children, bodyClass, isHome }) => {
                                 </div>
                             </div>
                             { isHome ?
+                                <div><div className="button-container"></div>
                                 <div className="site-banner">
-                                    <h1 className="site-banner-title">{site.title}</h1>
-                                    <p className="site-banner-desc">{site.subtitle}</p>
-                                </div> :
+                                    {/* empty div for space */}
+                                </div>
+                                <div className="site-banner">
+                                    <a href="/about" className="site-banner-title">About the project</a>
+                                </div>
```

Comment:
We should continue using the existing code here `site.title`, `site.subtitle`

Are we using react-helmet?

Reply:
@pi0neerpat Indeed react-helmet is in play

I pushed an updated before reading your review here, I'll readjust this to the original.